prog/weaver/http.go

```diff
@@ -19,6 +19,8 @@ import (
 	weave "github.com/weaveworks/weave/router"
 )
 
+var allConnectionStates = []string{"established", "pending", "retrying", "failed", "connecting"}
```

Comment:
Probably ignore this, as I've just spotted that the list existed before this PR.